syncd/SaiDiscovery.cpp

```diff
@@ -289,6 +297,15 @@ std::set<sai_object_id_t> SaiDiscovery::discover(
 {
     SWSS_LOG_ENTER();
 
+    return discover(&startRid, 1);
+}
+
+std::set<sai_object_id_t> SaiDiscovery::discover(
+        _In_ sai_object_id_t* rids,
+        _In_ size_t count)
+{
+    SWSS_LOG_ENTER();
+
```

Comment:
why here will be multiple rids to start ? call function in line 287 multiple times in your loop, don't put loop here

Reply:
discover(rid) changes log level. I find it somewhat slow (in comparison to all the gets) when called in a loop.
Changing log level in a loop is an unusual use case to optimize sai driver for, therefore prefer discover multiple at once. Multi rids version can also take adventage of bulk SAI calls if neccessary.